pilot/pkg/networking/core/v1alpha3/httproute.go

```diff
@@ -495,21 +493,11 @@ func BuildSidecarOutboundVirtualHosts(node *model.Proxy, push *model.PushContext
 			push.AddMetric(model.DuplicatedDomains, name, node.ID, msg)
 		}
 		if len(domains) > 0 {
-			perRouteFilters := map[string]*anypb.Any{}
```

Comment:
Should we leave both options ? This avoids backward incompat concerns.

Reply:
I can leave both options